village/src/App.js

```diff
@@ -11,14 +13,26 @@ class App extends Component {
       smurfs: [],
     };
   }
+  componentDidMount() {
+    axios
+      .get(`http://localhost:3333/smurfs`)
+      .then(response => {
+        console.log(response.data);
+        this.setState({ smurfs: response.data })
+      })
+    }
+    
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        <NavLink to="/">Smurfs </NavLink>
+        --<NavLink to="/smurf-list"> Add Smurf</NavLink>
+      
+        <Route path = '/smurf-list' render={(props) => <SmurfForm {...props} />}/>
+        <Route path = '/' render={(props) => <Smurfs {...props} smurfs={this.state.smurfs} />}/>
```

Comment:
Why format these like this when you don't consistently do so? Typically you'd expect no spaces around the `=` in a component:
```javascript
<Route path='/smurf-list' render={props => <SmurfForm {...props} />} />
<Route 
  path='/' 
  render={props => <Smurfs {...props} smurfs={this.state.smurfs} />}
/>
```